.github/workflows/showtime-trigger.yml

```diff
@@ -0,0 +1,106 @@
+name: 🎪 Superset Showtime
+
+# Ultra-simple: just sync on any PR state change
+on:
+  pull_request_target:
+    types: [labeled, synchronize, closed]
```

Comment:
`synchronize` can be dangerous.
- Attacker submits a PR, without issues
- Maintainer labels the PR and spins an ephemeral env
- Attacker pushes a new change that contains malicious code (XSS for example)
- sync kicks in and updates the ephemeral env with this code